migration-from-4x.md

```diff
@@ -0,0 +1,328 @@
+
+TODO
+
+#### New Settings
+
+TODO
+
+#### Removed Settings
+
+* **fixed_toolbar_container**: Owing to the enhancements to the new inline toolbar behaviour, `fixed_toolbar_container` is not required in TinyMCE 5.0.
+
+TODO
+
+### Methods
+
+* All TinyMCE 4.x methods for creating creating UI components have been removed. New methods have been added for TinyMCE 5.0. For more information, see the [docs]({{site.baseurl}}/components/).
+* No core editor methods were removed (tinymce, editor, selection, on(), etc remain the same).
+
+## Themes
+
+In TinyMCE 5.0, some themes have been removed and are now combined in a new single responsive theme called **Silver**.
+
+| **Removed Theme** | **Replaced by** |
+| ----------------- | --------------  |
+| [Modern](https://www.tiny.cloud/docs/themes/modern/) | Silver |
+| [Modern inline](https://www.tiny.cloud/docs/general-configuration-guide/use-tinymce-inline/) | Silver Inline |
+| [Inlite(Distraction-free Editor)](https://www.tiny.cloud/docs/themes/inlite/) | Silver (distraction free configuration) |
+| [Mobile](https://www.tiny.cloud/docs/themes/mobile/) | Silver (responsive to screen size)  |
+
+### Removed Themes
+
+#### Inlite
+
+The Inlite theme is no longer supported in TinyMCE 5.0. The features that the Inlite theme used to provide are now available as a plugin. For a workaround, use the following configuration:
+```
+{
+  theme: 'silver',
+  inline: true,
+  toolbar: false,
+  plugins: [ 'inlite' ]
+}
+ ```
+This will provide a similar but improved distraction free experience in TinyMCE 5.0.
+
+#### Modern
+
+The Modern theme is no longer supported in TinyMCE 5.0.  The modern themes Ui library `tinymce.ui.*` has also been deleted. This change may impact integrations depending upon the level of customization.
+
+For changes required, refer to the following table of examples:
+
+| Customization Level | Description | Impact |
+| ------------------- | ----------- | ------ |
+| Minor | Some custom buttons | no Ui fixes required, update `addButton` configuration to TinyMCE 5.0 format |
+| Moderate | A dialog created using `editor.windowManager.open` configuration objects | Convert TinyMCE 4.x config to TinyMCE 5.0 config |
+| Major | Completely custom dialogs and extended use of the Modern UI framework | Not all API use cases are covered by our new TinyMCE 5.0 components. However, we will strive to create supported workarounds or if there are sufficient requests, add a new component to resolve the use case. |
+
+> Note: Please provide feedback on your use case and your Tinymce 4.x configuration containing only the UI component that you wish to be supported or need a workaround.
+
+> Note: The Silver theme in TinyMCE 5.0 contains a set of configurable UI components that could be used to replace the current customizations (modern, inline, and inlite theme). Silver is enabled by default. If you do not specify a theme, it will default to Silver.
```

Comment:
The first sentence of this note probably needs clarification. To me, it seems unrelated to the rest of the paragraph at the moment?

Reply:
Reviewed and changed content.